styles/desk-style.css

```diff
@@ -0,0 +1,185 @@
+
+/* This applies from 600px onwards */
+@media (min-width: 992px) {
+  /* Header of the page */
+  .portfo-header {
+    margin-bottom: 40px;
+    margin-right: 50px;
+  }
+
+  .desk-nav {
+    display: flex;
+    justify-content: space-around;
+  }
+
+  #hamburger-btn {
+    display: none;
+  }
+
+  .desk-nav li {
+    padding-right: 20px;
+  }
+
+  .desk-nav li a {
+    color: #344563;
+  }
+
+  /* Start hero section */
+  .hero {
+    background-image: url("../images/header-bg-des.svg");
+    margin-bottom: 50px;
+  }
+
+  .hero-content {
+    margin-top: 5%;
+    margin-left: 18%;
+    margin-right: 15%;
+  }
+
+  .hero-content h1 {
+    font-size: 48px;
+  }
+
+  .hero-content p {
+    font-size: 20px;
+  }
+
+  .snapshoot {
+    display: none;
+  }
+
+  .snapshoot-div {
+    width: 544px;
+    height: 448px;
+  }
+
+  .desk-snapshoot {
+    width: 100%;
+    height: 100%;
+    padding-top: 30px;
+    display: block;
+  }
+
+  .work {
+    margin-left: auto;
+    margin-right: auto;
+    width: 1156px;
+  }
```

Comment:
- [ ] There is too much space than it needs to be between the content of the work card. Please see the images below and decrease spacing in 3 places as I showed you 🤗

#### Your design
![image](https://user-images.githubusercontent.com/69011963/150568532-09a186a2-e816-436e-ae41-d5e37099a005.png)